lib/active_record/connection_adapters/sqlserver/schema_statements.rb

```diff
@@ -281,6 +281,21 @@ def create_schema_dumper(options)
           SQLServer::SchemaDumper.create(self, options)
         end
 
+        def create_schema(schema_name, authorization = nil)
+          sql = "CREATE SCHEMA [#{schema_name}]"
+          sql += " AUTHORIZATION [#{authorization}]" if authorization
+
+          execute sql
+        end
+
+        def alter_schema(schema_name, table_name)
```

Comment:
I'm not sure if `alter_schema` is the best name here.

How about `change_table_schema(table_name, schema_name)` like there's [`change_table_comment`](https://apidock.com/rails/v6.0.0/ActiveRecord/ConnectionAdapters/SchemaStatements/change_table_comment) for example?